contracts/IAM.sol

```diff
@@ -0,0 +1,79 @@
+pragma solidity ^0.5.17;
+
+import "./CampaignFactory.sol";
+
+contract IAM {
+    address owner;
+    // CampaignFactory factoryContract;
+
+    // the NONE status maps to 0; for unmapped addresses
+    enum status { NONE, VERIFIED, LOCK, DISTRUST }
+    mapping(address => status) orgStatus;
+    address[] orgList;
+    
+    event addVerifiedOrg(address org);
+    event orgVerified(address org);
+    event orgLocked(address org);
+    event orgDistrust(address org);
+
+    constructor() public {
+        owner = msg.sender;
+    }
+
+    // --- MODIFIERS ---
+    modifier ownerOnly() {
+        require(owner == msg.sender);
+        _;
+    }
+
+    // --- GETTERS / SETTERS ---
+    // may be used later
+    function isVerified(address organisation) public view returns (bool) {
+        return (orgStatus[organisation] == status.VERIFIED);
+    }
+
+    function isLocked(address organisation) public view returns (bool) {
+        return (orgStatus[organisation] == status.LOCK);
+    }
+
+    function isDistrust(address organisation) public view returns (bool) {
+        return (orgStatus[organisation] == status.DISTRUST);
+    }
+
+    // function setFactory(CampaignFactory factoryAddress) public ownerOnly {
+    //     require(address(factoryAddress) != address(0));
+    //     factoryContract = factoryAddress;
+    // }
```

Comment:
This is for transferring ownership of the Factory is it?

Reply:
This was originally meant for potential callbacks, removed for clarity.